torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1020,6 +1111,50 @@ def sample_inputs_pinverse(op_info, device, dtype, requires_grad=False):
                                 dtypes=[torch.bfloat16])),
                    promotes_integers_to_float=True,
                    handles_complex_extremals=False),
+    UnaryUfuncInfo('abs',
```

Comment:
Adding these OpInfos is great but we should separate their addition. 

Maybe the OpInfo for abs can go in the PR adding alias support to OpInfos? (That PR can also update test_op_aliases.py.)